tests/unit/migrations/namespace-form-fields.js

```diff
@@ -4,6 +4,37 @@ var sinon = require('sinon'),
     utils = require('../utils'),
     migration = require('../../../migrations/namespace-form-fields');
 
+// Arguments : docArrayForFirstBatch, docArrayForSecondBatch, ...
```

Comment:
You don't need this comment anymore right? You can see what to pass because you have a parameter